qnv.py

```diff
@@ -36,7 +36,7 @@ def quatMultiplyUnnorm(q1,q2): #returns quaternion product (product is not a uni
 	q = np.hstack((a,b))
 	return q
 
-def quatRotate(q,x): #rotates vecctor x by quaternion q
+def quatRotate(q,x): #rotates vecctor x by quaternion q (According to our convention, q rotates from inertial to body)

```

Comment:
Not necessarily. If we give q_IB as an input then it will rotate x to inertial frame assuming it originally being in body frame